olive/systems/azureml/aml_evaluation_runner.py

```diff
@@ -42,12 +42,12 @@ def main(raw_args=None):
 
     # load model
     model_config = get_model_config(common_args)
-    model = ModelConfig.from_json(model_config).create_model()
-
     # load metric
     metric_config = json.load(open(metric_args.metric_config))
     metric = create_metric(metric_config, metric_args)
 
+    model = ModelConfig.from_json(model_config).create_model()
```

Comment:
nit: is this just a leftover from undone changes?

Reply:
Ah yeah I moved this line from before metric to after metric since I was going to update hfconfig for model config and metric config both.